tests/data/qasm/sleep.qasm

```diff
@@ -0,0 +1,17 @@
+OPENQASM 2.0;
```

Comment:
Good minimal test. I'd like to see an additional program use multiple sleeps on different qubits just as an extra sanity check in the test suite.